### ISSUE TITLE
Add note for <select multiple> keyboard support across browsers

### ISSUE THREAD

**Issue by charmarkk** [mdn/sprints#1504]
# Request type

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [x] Correction or update

# Details
Hi! I'm wondering if the documentation `<select multiple>` ([link to select multiple](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select)) should be updated or a note created to reflect issues with keyboard support? Currently, only Firefox allows full keyboard support, as in Ctrl-Arrow and Space to select multiple, non-contiguous items. See this [ARIA authoring practices discussion](https://github.com/w3c/aria-practices/issues/919) for some comments.

**Comment by chrisdavidmills** [comment]
Hi @charmarkk ! I've finally got round to dealing with this; I added a section to the `<select>` page to cover ways of selecting multiple items:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#Selecting_multiple_options

Hope this sounds OK to you. 

Also worth noting is that on macOS, even when you've disabled the conflicting OS shortcuts, the non-contiguous item select doesn't work properly — pressing the space bar to select the focused item doesn't seem to work. I've filed a bug about this at https://bugzilla.mozilla.org/show_bug.cgi?id=1558444.

**Comment by charmarkk** [comment]
Hey Chris! My thanks for writing up the note. I think it sounds good, but I wonder if we might sort of flip the layouts and be a little tougher on the note? Something like this ( understanding this was 1) a simple DOM edit and 2) I don't know _all_ the ins and outs of MDN's formatting :) ):

![image](https://user-images.githubusercontent.com/22264269/59289814-93f8e700-8c3c-11e9-90a8-5c81462a5f37.png)

Image Alt: Shifted warning note up above keyboard section; changed "note" in original page in Chris's issue to read "warning"; solidified wording that this pattern only works in Firefox atm.

I say that mostly because I worry that implementers looking to MDN for guidance might not get as far as the note, and go ahead and use the pattern regardless. That would create a huge barrier for keyboard only users, whether or not they have a disability. Of course, I understand if it's better to stick with convention. Just some thoughts! :)

**Comment by chrisdavidmills** [comment]
@charmarkk I think you've got a good point here. I've moved the note up as suggested, and changed it to a "warning" instead (slightly scarier red box)

**Comment by charmarkk** [comment]
Cool cool, thanks for the edits there. I think that looks really good, and hopefully the browser devs can fix that sometime soon! Also, good find on the MacOS problem, I'm not sure that even we (over at APG) found that one. 😄 

Many thanks @chrisdavidmills! I think that should solve this one. 🙂

**Comment by chrisdavidmills** [comment]
Cool, good working with you! I'll close this one then.

**Comment by charmarkk** [comment]
@chrisdavidmills Sorry to bug you again, but just noticed. The warning says "multiple **non-contiguous** items" - it should probably just read "multiple items." 🙂

**Comment by chrisdavidmills** [comment]
@charmarkk As an FYI, the Fx macOS bug I reported now seems to have gone away; seems like it was a short-lived regression or something. So we are working on mac too now!

**Comment by charmarkk** [comment]
@chrisdavidmills Good to know! I'll check it out and see if I can add some positivity to the APG issue we have open for this. Thanks for checking it out!
